lib/intacct/models/project.rb

```diff
@@ -144,7 +145,95 @@ def create_xml(xml)
       end
 
       def update_xml(xml)
-        create_xml(xml)
+        xml.PROJECTID attributes.projectid
+        xml.NAME attributes.name
+        xml.DESCRIPTION attributes.description if attributes.description.present?
+        xml.PARENTID attributes.parentid if attributes.parentid.present?
+        xml.INVOICEWITHPARENT attributes.invoicewithparent if attributes.invoicewithparent.present?
+        xml.PROJECTCATEGORY attributes.projectcategory
+        xml.PROJECTTYPE attributes.projecttype if attributes.projecttype.present?
+        xml.PROJECTSTATUS attributes.projectstatus if attributes.projectstatus.present?
+        xml.CUSTOMERID attributes.customerid if attributes.customerid.present?
+        xml.MANAGERID attributes.managerid if attributes.managerid.present?
+        xml.CUSTUSERID attributes.custuserid if attributes.custuserid.present?
+        xml.SALESCONTACTID attributes.salescontactid if attributes.salescontactid.present?
+        xml.BEGINDATE attributes.begindate.strftime("%m/%d/%Y") if attributes.begindate.present?
+        xml.ENDDATE attributes.enddate.strftime("%m/%d/%Y") if attributes.enddate.present?
+        xml.DEPARTMENTID attributes.departmentid if attributes.departmentid.present?
+        xml.LOCATIONID attributes.locationid if attributes.locationid.present?
+        xml.CLASSID attributes.classid if attributes.classid.present?
+        xml.CURRENCY attributes.currency if attributes.currency.present?
+        xml.BILLINGTYPE attributes.billingtype if attributes.billingtype.present?
+        xml.TERMNAME attributes.termname if attributes.termname.present?
+        xml.DOCNUMBER attributes.docnumber if attributes.docnumber.present?
+        xml.BILLTO attributes.billto if attributes.billto.present?
+        xml.SHIPTO attributes.shipto if attributes.shipto.present?
+        xml.CONTACTINFO attributes.contactinfo if attributes.contactinfo.present?
+        xml.SONUMBER attributes.sonumber if attributes.sonumber.present?
+        xml.PONUMBER attributes.ponumber if attributes.ponumber.present?
+        xml.POAMOUNT attributes.poamount if attributes.poamount.present?
+        xml.PQNUMBER attributes.pqnumber if attributes.pqnumber.present?
```

Comment:
Can we not turn this into some type of mapping because it seems like it is all very similar code?

Reply:
Maybe. This file is generated so maybe we can simplify the generation in the future but it's not worth the effort at the moment.